API.md

```diff
@@ -442,6 +442,31 @@ Proxy to Home Assistant Core websocket.
 }
 ```
 
+### Network
+
+- GET `/network/info`
+
+Get network information
+
+```json
+{
+  "primary connection": {
+    "ip_address": "192.168.2.149",
+    "Gateway": "192.168.2.1",
+    "Id": "Wired connection 1",
+    "Type": "802-3-ethernet"
+  },
+  "connections": [
+    {
+      "ip_address": "192.168.2.149",
+      "Gateway": "192.168.2.1",
+      "Id": "Wired connection 1",
+      "Type": "802-3-ethernet"
+    }
+  ]
+}
+```
```

Comment:
Let's simplify this (also for CLI):
```
{
  "connections": [
    {
      "ip_address": "192.168.2.149",
      "gateway": "192.168.2.1",
      "id": "Wired connection 1",
      "type": "802-3-ethernet",
      "primary": "bool"
    }
  ]
}
```

Reply:
I know networkmanager handle it different, but we don't need follow exactly his schema, we can map this between our network layer and the networkmanager dbus.

Reply:
In that case, just drop the "connection" and just serve a list I guess

Reply:
Looks like a dict is required for it to return anything :shrug: